src/lib/mdns/Resolver_ImplMinimalMdns.cpp

```diff
@@ -238,11 +223,26 @@ void PacketDataReporter::OnResource(ResourceType type, const ResourceData & data
         }
         else if (mDiscoveryType == DiscoveryType::kOperational)
         {
-            OnOperationalSrvRecord(data.GetName(), srv);
+            // Ensure this is our record.
```

Comment:
nit: reword comment? I am unsure if this helps provide more context or information beyond reading the code.